app/src/main/java/com/arctouch/movies/serverapi/ClientApiConfig.java

```diff
@@ -0,0 +1,57 @@
+package com.arctouch.movies.serverapi;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import okhttp3.Interceptor;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public class ClientApiConfig {
```

Comment:
Nothing in this class is ever used... Why does it exist?

Reply:
This is my initial code structure for any project :) ... Thinking for next versions